### PR TITLE
Shuffle trigger state into pending when report to github

### PR DESCRIPTION
**Pull request by krzyzacy**
looks like plank handles description already in https://github.com/kubernetes/test-infra/blob/91186ea67814300ed42df1188bdd2357d1d86bb2/prow/plank/controller.go#L466-L473

/assign @cjwagner

### CODE REVIEW

File: prow/report/report_test.go [hunk 1]
```diff
@@ -322,6 +322,14 @@ func TestReportStatus(t *testing.T) {
 
 			expectedStatuses: []string{"failure"},
 		},
+		{
+			name: "triggered_job-report_true",
```
[the remark this answers is not in the document]
Comment (krzyzacy) [reply prow/report/report_test.go]: lol, sure

**Comment by stevekuznetsov** [comment]
@krzyzacy <3
/lgtm
/hold

for @cjwagner

File: prow/report/report.go [hunk 2]
```diff
@@ -45,20 +45,39 @@ type GithubClient interface {
...
+	case v1.PendingState:
+		return github.StatusPending, nil
+	case v1.SuccessState:
+		return github.StatusSuccess, nil
+	case v1.ErrorState:
+		return github.StatusError, nil
+	case v1.FailureState:
+		return github.StatusFailure, nil
+	case v1.AbortedState:
+		return github.StatusFailure, nil
```
Comment (cjwagner) [inline prow/report/report.go]: It is unclear to me if this should be marked as a failure or an error, but I don't think it matters too much.

**Pull request merged**